CHANGELOG.md

```diff
@@ -7,6 +7,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.3.4] - 2024-08-27
```

Comment:
Given that `0.4.0` went in but hasn't been released yet, should we include these changes in the same version too?